docs/prover/cli/options.md

```diff
@@ -687,3 +687,12 @@ The `solver spec` can be a single solver (`-solver=z3`) or a list of solvers
 example, `cvc5` refers to the default configuration of `cvc5` whereas
 `cvc5:nonlin` is better for nonlinear problems.  Additional options can be set
 via `z3{randomSeed=17}`.
+
+(-useBitVectorTheory)=
+#### `--settings -useBitVectorTheory`
+
+This option reduces the Prover {term}`overapproximation`s used for bitwise
```

Comment:
Does the word "reduces" here have a technical meaning that provides context? If not, it could be helpful to rephrase slightly to make clearer what is meant by "reduces the...overapproximations" for people with a different technical background. 

If there is a technical meaning at play, an inline synonym could be helpful.